libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -441,6 +441,7 @@ singleton AP_ESC_Telem method get_usage_seconds boolean uint8_t 0 NUM_SERVO_CHAN
 singleton AP_ESC_Telem method update_rpm void uint8_t 0 NUM_SERVO_CHANNELS uint16_t'skip_check float'skip_check
 singleton AP_ESC_Telem method update_telem_data void uint8_t 0 NUM_SERVO_CHANNELS AP_ESC_Telem_Backend::TelemetryData uint16_t'skip_check
 singleton AP_ESC_Telem method set_rpm_scale void uint8_t 0 NUM_SERVO_CHANNELS float'skip_check
+singleton AP_ESC_Telem method get_last_telem_data_ms uint32_t uint8_t 0 NUM_SERVO_CHANNELS
```

Comment:
```suggestion
singleton AP_ESC_Telem method get_last_telem_data_ms uint32_t uint8_t 0 ESC_TELEM_MAX_ESCS
```

Reply:
In this case shouldn't all the other ESC_telem defines be change from NUM_SERVO_CHANNELS to ESC_TELEM_MAX_ESCS

Reply:
> In this case shouldn't all the other ESC_telem defines be change from NUM_SERVO_CHANNELS to ESC_TELEM_MAX_ESCS

Possibly?  I based this suggestion on the sanity check in the top of that method, nothing more.